packages/angular_devkit/build_webpack/src/index.ts

```diff
@@ -11,4 +11,4 @@ export * from './webpack-dev-server';
 
 export * from './plugins/architect';
 
-export { EmittedFiles } from './utils';
+export { getEmittedFiles, EmittedFiles } from './utils';
```

Comment:
This is intended to be private. You need to get the emitted files, from the compilation result.